javascript/ql/src/experimental/semmle/javascript/Actions.qll

```diff
@@ -0,0 +1,309 @@
+/**
+ * Libraries for modelling GitHub Actions workflow files written in YAML.
+ * See https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions.
+ */
+
+import javascript
+
+/**
+ * Libraries for modelling GitHub Actions workflow files written in YAML.
+ * See https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions.
+ */
+module Actions {
+  /** A YAML node in a GitHub Actions workflow file. */
+  private class Node extends YAMLNode {
+    Node() { this.getLocation().getFile().getRelativePath().matches(".github/workflows/%") }
```

Comment:
If I modify the pattern to "%.github/workflows/%" it makes tests pass, but the query is more precise when the ".github" must be in the root.  The way GitHub actions work workflows are picked only from the root ".github/workflows" folder.

How to make it work with tests and do not sacrifice the correctness of the query?